languages/elixir/exercises/concept/exceptions/.docs/introduction.md

```diff
@@ -0,0 +1,52 @@
+All errors in Elixir implement the _Exception Behaviour_. Just like the _Access Behaviour_, the _Exception Behavior_ defines callback functions that a module must implement to fulfill the software contract of the behaviour. Once an errors is defined, it has the following properties:
+
+- The module's name defines the error's name.
+- The module defines an error-struct.
+- The module will have a `:message` field.
+- The module can be be used with `raise/1` and `raise/2` to raise the intended error
+
+The _Exception Behaviour_ also specifies two required callbacks: `message/1` and `exception/1`. Despite being required, `message/1` is often left to be implemented by the _behavior_, but it is common to implement `exception/1` instead to get custom error messages.
+
+## Defining an exception
+
+To define an exception from an error module, we use the `defexception` macro function:
+
+```elixir
+# Defines a minimal error, with the name `MyError`
+defmodule MyError do
+  defexception [:message]
+end
+
+# Defines an error with a customized exception/1 function
+defmodule MyCustomizedError do
+  defexception [:message]
+
+  @impl true
```

Comment:
Is the `@impl true` bit a piece of syntax that is introduced earlier in the track? Just curious :)

Reply:
I think it's mentioned in the access exercsise, if not it should be mentioned there.

Reply:
Okay great!